env0/utils.go

```diff
@@ -230,10 +230,16 @@ func writeResourceData(i interface{}, d *schema.ResourceData) error {
 
 		switch fieldType.Kind() {
 		case reflect.String:
+			if field.String() == "" {
```

Comment:
This is a little naive. But assume if empty - skip (don't set).

Reply:
are you sure? 
don't we have cases where this is a legit value? (maybe it is wrong on the BW side but we still need to support this case if that happen)

Reply:
@razbensimon - you may review the additional changes in https://github.com/env0/terraform-provider-env0/pull/432/commits/accbc9be68f7af3bc3d81abcba96d70e2382df2b